sql/src/main/java/io/crate/execution/ddl/RerouteActions.java

```diff
@@ -86,6 +93,27 @@ private RerouteActions() {
         return listener;
     }
 
+    public static ClusterRerouteRequest prepareAllocateStalePrimaryReq(PromoteReplicaStatement promoteReplica,
```

Comment:
would it make sense to name this `preparePromoteStalePrimaryReq`? (to be in line with the statement)

Reply:
yep, I'll sneak in the rename in another PR I already started working on where I touch this class already to avoid having to resolve conflicts in a rebase.